README.md

```diff
@@ -131,6 +134,15 @@ To build the app for production, run:
 yarn build:client
 ```
 
+NOTE: By default the demo wallet operates only on Stellar testnet network. To
+use mainnet or any other Stellar network, add below to the client **.env**
+(example for mainnet):
+
+```
+REACT_APP_HORIZON_PASSPHRASE = "Public Global Stellar Network ; September 2015"
```

Comment:
Shouldn't this be `REACT_APP_NETWORK_PASSPHRASE`?

Reply:
Otherwise LGTM

Reply:
The idea was to group them together by `REACT_APP_HORIZON_` and `REACT_APP_HORIZON_NETWORK_PASSPHRASE` seemed too long. We can change it to `REACT_APP_NETWORK_PASSPHRASE` if it will be more clear.

Reply:
Got it. I don't feel strongly about it so we can keep it as-is.